crates/viewer/re_ui/src/design_tokens.rs

```diff
@@ -85,6 +85,11 @@ impl DesignTokens {
             ..Default::default()
         };
 
+        // We preserve the text cursor blink. It's false by default if we're in a test
+        // and true by default otherwise. We don't want to interfere with this behavior
+        // (Having a blinking cursor in tests can make the tests flaky).
+        egui_style.visuals.text_cursor.blink = ctx.style().visuals.text_cursor.blink;
+
```

Comment:
I think this would be more robust, in case there are (or will be) other things set special for tests (or other things)

```suggestion
let mut egui_style = Arc::unwrap_or_clone(ctx.style_of(egui::Theme::Dark));
```